hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassDefinitionsServiceImpl.java

```diff
@@ -533,6 +698,64 @@ public Trigger updateTrigger(String tenantId, Trigger trigger) throws Exception
         return trigger;
     }
 
+    @Override
+    public Trigger orphanMemberTrigger(String tenantId, String memberId) throws Exception {
+        if (isEmpty(tenantId)) {
+            throw new IllegalArgumentException("TenantId must be not null");
+        }
+        if (isEmpty(memberId)) {
+            throw new IllegalArgumentException("TriggerId must be not null");
+        }
+
+        checkTenantId(tenantId, memberId);
```

Comment:
In this case checkTenantId is not needed. This helper method is used to ensure that Trigger/Dampening/Tag objects have tenantId correctly. When arguments is different of that, it can be omitted.